csg.js

```diff
@@ -1983,11 +1983,11 @@ for solid CAD anyway.
                                 this._z = 0;
                             }
                         }
-                    } else if (('_x' in x) && ('_y' in x)) {
-                        this._x = parseFloat(x._x);
-                        this._y = parseFloat(x._y);
-                        if ('_z' in x) {
-                            this._z = parseFloat(x._z);
+                    } else if (('x' in x) && ('y' in x)) {
```

Comment:
Ok , do you mean this one ?
https://github.com/z3dev/csg.js/blob/207c06310244937436d8c5f3c088c7ac457ace80/csg.js#L6422 ? so this fix would allow passing the object directly, without the need for obj.pos._x,obj.pos._y ?